packages/babel-preset-env/test/fixtures/preset-options/safari-tagged-template-literals/output.js

```diff
@@ -1,5 +1,2 @@
-var _ = function _(t) {
-    return t;
-  },
-  _t;
-tag(_t || (_t = _`Safari 12 borked`));
+var _templateObject;
+tag(_templateObject || (_templateObject = babelHelpers.taggedTemplateLiteral(["Safari 12 borked"])));
```

Comment:
This doesn't seem right

Reply:
The template literals compat data is bumped because of https://github.com/compat-table/compat-table/pull/1966, for the time being we can ignore this sub-test. In the future we should test if there is invalid escape in tagged template, and apply the plugin when required.